### DISCUSSION TITLE
Cross-Referencing a Tip fails！

### DISCUSSION THREAD

**Discussion by wisejky** [quarto-dev/quarto-cli#9022] [Q&A] [answered]
```qmd
---
title: "Untitled"
format: html
---

::: {#tip-example .callout-tip}
## Cross-Referencing a Tip

Add an ID starting with `#tip-` to reference a tip.
:::

See @tip-example...

```

<img width="433" alt="cross ref fail" src="https://github.com/quarto-dev/quarto-cli/assets/120883966/949c4efc-206a-4f46-928c-82db953bef2e">

**Comment by mcanouil** [comment]
I cannot reproduce. Which version of Quarto are you using?

<img width="494" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/7eaa52d6-c80a-4546-a237-3aab2cd676d9">

**Reply by wisejky** [reply]
thanks for your reply！
i use Rstudio ，and 1.4 quarto

i also test it on my second PC，the problem is the same

**Reply by wisejky** [reply]
oh i check it again，maybe we can use 1.4 quarto on Rstudio until 2024.04+version？

so now the cross ref tip only available  for terminal？

thanks for your “tips”

**Reply by mcanouil** [reply, marked as answer]
I tested on 1.5.

**Reply by wisejky** [reply]
well， i done，thanks！！

**Comment by cderv** [comment]
FWIW this is working fine on Quarto 1.4 too 
![image](https://github.com/quarto-dev/quarto-cli/assets/6791940/7f25c187-82bb-49a3-9aa6-2579d51a1227)

This is screenshot of a HTML with 1.4.551 
````html
<meta name="generator" content="quarto-1.4.551">
````

@wisejky you can check with which version your document is produced by looking in the HTML source and the `<head>` for this `<meta>`
